lib/tasks/pubmed.rake

```diff
@@ -10,4 +10,36 @@ namespace :pubmed do
     doc = client.fetch_records_for_pmid_list(pmids)
     puts doc # XML document
   end
+
+  desc 'Harvest using a text file with a list of publications by PubmedID for the supplied cap_profile_id'
+  task :pmid_profile_id_import, [:path_to_report,:cap_profile_id] => :environment do |_t, args|
```

Comment:
i agree the current name is a bit clunky, but I named it this way on purpose for consistency with the corresponding and similar WOS ID import here https://github.com/sul-dlss/sul_pub/blob/master/lib/tasks/sw.rake#L58